sdk/spring/spring-reference.yml

```diff
@@ -97,7 +97,7 @@
               }
           springProperties:
             starter: false
-            compatibilityRange: "[2.5.5,2.6.2)"
```

Comment:
I am not sure about the format here, you can ask @yiliuTo to confirm.

Reply:
It could be "]" as long as we support 2.6.2